osu.Game/Overlays/MusicController.cs

```diff
@@ -273,20 +273,21 @@ private void beatmapChanged(WorkingBeatmap beatmap)
             {
                 bool audioEquals = beatmapBacking.Value?.BeatmapInfo?.AudioEquals(current.BeatmapInfo) ?? false;
 
-                if (audioEquals)
-                    direction = TransformDirection.None;
-                else if (queuedDirection.HasValue)
+                if (!audioEquals)

```

Comment:
Move audioEquals out of this if, and make the conditional just `if (!audioEquals)` to reduce nesting.